controller/app.controller.js

```diff
@@ -0,0 +1,9 @@
+const { gettingTopics } = require('../modele/app.model')
+
+exports.getTopics = (req, res) => {
+    
+    return gettingTopics().then((topics) => {
+       
+     return   res.status(200).send(topics)
+    })
```

Comment:
Generally we want to send data back on a relevant key in an object where as this is just the data currently. Add this to a key of topics.